DataManagementSystem/Agent/FTS3Agent.py

```diff
@@ -1,5 +1,34 @@
 """
-  FTS3Agent implementation
+ .. versionadded:: v6r20
+
+  FTS3Agent implementation.
+  It is in charge of submitting and monitoring all the transfers. It can be duplicated.
+
+
+::
+
+  FTS3Agent
+  {
+    PollingTime = 120
+    MaxThreads = 10
+    # How many Operation we will treat in one loop
+    OperationBulkSize = 20
+    # How many Job we will monitor in one loop
+    JobBulkSize = 20
+    # Max number of files to go in a single job
+    MaxFilesPerJob = 100
+    # Max number of attempt per file
+    maxAttemptsPerFile = 256
```

Comment:
For consistency, could we start this option also with a capital letter?

Reply:
Absolutely